chapters/synchronous.tex

```diff
@@ -1188,37 +1200,35 @@ \subsection{Solver Methods}\label{solver-methods}
 If a tool does not support the integration method defined in a model, typically a warning message is printed and the method is changed to \lstinline!"External"!.
 \end{nonnormative}
 
-If the solver method is \lstinline!"External"!, then the partition associated with this method is integrated by the simulation environment for an interval of length of
-\lstinline!interval()! using a solution method defined in the simulation environment.
+If the solver method is \lstinline!"External"!, then the sub-partition associated with this method is integrated by the simulation environment for an interval of length of \lstinline!interval()! using a solution method defined in the simulation environment.
 
 \begin{nonnormative}
-An example of such a solution method could be to have a table of the clocks that are associated with discretized continuous-time partitions and a method selection per clock.
+An example of such a solution method could be to have a table of the clocks that are associated with discretized sub-partitions and a method selection per clock.
 In such a case, the solution method might be a variable step solver with step-size control that integrates between two clock ticks.
 The simulation environment might also combine all partitions associated with method \lstinline!"External"!, as well as all continuous-time partitions, and integrate them together with the solver selected by the simulation environment.
 \end{nonnormative}
 
-If the solver method is \emph{not} \lstinline!"External"!, then the partition is
-integrated using the given method with the step-size \lstinline!interval()!.
+If the solver method is \emph{not} \lstinline!"External"!, then the sub-partition is integrated using the given method with the step-size \lstinline!interval()!.
 
 \begin{nonnormative}
 For a periodic clock, the integration is thus performed with fixed step size.
 \end{nonnormative}
 
-The solvers are defined with respect to the underlying ordinary
-differential equation in state space form to which the continuous-time
-partition can be transformed, at least conceptually ($t$ is time,
-$u_{c}(t)$ is the continuous-time \lstinline!Real! vector
-of input variables, $u_{d}(t)$ is the
-discrete-time \lstinline!Real!/\lstinline!Integer!/\lstinline!Boolean!/\lstinline!String! vector of input variables,
-$x(t)$ is the continuous-time real vector of states, and
-$y(t)$ is the continuous-time or discrete-time
-\lstinline!Real!/\lstinline!Integer!/\lstinline!Boolean!/\lstinline!String! vector of algebraic and/or output variables):
+The solvers are defined with respect to the underlying ordinary differential equation in state space form to which the continuous-time sub-partition can be transformed, at least conceptually:
```

Comment:
I don't like the use of "continuous-time sub-partition" here because we have not defined such a thing. I am guessing it is discretized sub-partition before it is discretized. But I do not know what would be a good term for it.

Reply:
Type it out: "discretized sub-partition before it is discretized"

Reply:
```suggestion
The solvers are defined with respect to the underlying ordinary differential equation in state space form that corresponds to the sub-partition before it has been discretized, at least conceptually:
```